src/server/api/controllers/user.controller.js

```diff
@@ -0,0 +1,113 @@
+const knex = require('../../config/db');
+const Error = require('../lib/utils/http-error');
+
+const getUserById = async (id, role) => {
+  try {
+    const eventUsers = await knex.raw(
+      'SELECT ' +
+        'ev.id AS eventId, ' +
+        'org.name AS organization, ' +
+        'ev.event_date AS eventDate, ' +
+        'ev.created_at AS eventTime, ' +
+        'WEEK(ev.created_at) as eventWeek ' +
+        'FROM ' +
+        'events_users AS eu, ' +
+        'users AS us, ' +
+        'events_students AS es, ' +
+        'organizations AS org, ' +
+        'events AS ev ' +
+        'WHERE ' +
+        'us.id = eu.users_id ' +
+        'AND es.student_id = eu.id ' +
+        'AND org.id = us.organization_id ' +
+        'AND ev.id = es.event_id ' +
+        'AND eu.users_id = ? ' +
+        'limit 1;',
+      [id],
+    );
+    if (eventUsers.length === 0) {
+      return new Error(`No Entries with the id of ${id}`, 404);
+    }
+    const usableData = eventUsers[0][0];
+    usableData.eventDate = getDate(usableData.eventTime);
+    usableData.eventTime = String(usableData.eventTime).substr(16, 8);
+    const attendeeList = await getAttendees(role, usableData.eventId);
+    usableData.listOfAttendess = attendeeList;
+    return usableData;
+  } catch (error) {
+    return error.message;
+  }
+};
+
+async function getAttendees(role, eventId) {
+  if (role !== 'mentors' && role !== 'students') {
+    return new Error(`No Roles found with ${role}`, 404);
+  }
+  try {
+    if (role === 'mentors') {
+      const studentNames = await knex.raw(
+        'SELECT ' +
+          'us.name ' +
+          'FROM ' +
+          'users us, ' +
+          'events_students es, ' +
+          'events_users eu, ' +
+          'events ev ' +
+          'WHERE ' +
+          'ev.id = es.event_id ' +
+          'AND es.student_id = eu.id ' +
+          'AND eu.users_id = us.id ' +
+          'AND ev.id = ? ',
+        [eventId],
```

Comment:
Same as above, use ` to have a multiline string instead of using + all the time.